src/components/DailyStatistics/DailyStatisticsScreen.tsx

```diff
@@ -0,0 +1,187 @@
+import { NavigationProps } from "../../types";
+
+const styles = StyleSheet.create({
+  content: {
+    padding: size(2),
+    paddingVertical: size(6),
+    height: "100%",
+    width: 512,
+    maxWidth: "100%"
+  },
+  headerText: {
+    marginBottom: size(4)
+  },
+  bannerWrapper: {
+    marginBottom: size(1.5)
+  },
+  categoryName: {
+    fontFamily: "brand-bold",
+    fontSize: fontSize(1),
+    marginBottom: size(3)
+  }
+});
+
+const DailyStatisticsScreen: FunctionComponent<NavigationProps> = ({
+  navigation
+}) => {
+  useEffect(() => {
+    Sentry.addBreadcrumb({
+      category: "navigation",
+      message: "DailyStatisticsScreen"
+    });
+  }, []);
+
+  const messageContent = useContext(ImportantMessageContentContext);
+  const { config } = useConfigContext();
+  const showHelpModal = useContext(HelpModalContext);
+  const { sessionToken, endpoint, operatorToken } = useContext(AuthContext);
+  const { totalCount, setTotalCount } = useContext(StatisticsContext);
+  const {
+    currentTimestamp,
+    setCurrentTimestamp,
+    lastTransactionTime,
+    setLastTransactionTime,
+    transactionHistory,
+    setTransactionHistory
+  } = useContext(StatisticsContext);
+  const { showAlert } = useContext(AlertModalContext);
+  const { policies } = useContext(CampaignConfigContext);
+  const [error, setError] = useState<Error>();
+  const clearError = useCallback((): void => setError(undefined), []);
+
+  const fetchDailyStatistics = async (
+    currentTimestamp: number
+  ): Promise<void> => {
+    try {
+      const response = await getDailyStatistics(
+        currentTimestamp,
+        sessionToken,
+        endpoint,
+        [operatorToken]
+      );
+
+      const {
+        summarisedTransactionHistory,
+        summarisedTotalCount
+      } = summariseTransactions(response, policies);
+
+      setTransactionHistory(summarisedTransactionHistory);
+      setTotalCount(summarisedTotalCount);
+      setCurrentTimestamp(currentTimestamp);
+
+      if (response.pastTransactions.length !== 0) {
+        setLastTransactionTime(response.pastTransactions[0].transactionTime);
+      } else {
+        setLastTransactionTime(0);
+      }
+    } catch (error) {
+      setError(error);
+      showAlert({
+        ...systemAlertProps,
+        description: ERROR_MESSAGE.SERVER_ERROR,
+        onOk: () => {
+          navigateHome(navigation);
+          clearError();
+        }
+      });
+    }
+  };
+
+  const onPressPrevDay = (): void => {
+    const prevDay = getTime(subDays(currentTimestamp, 1));
+    setCurrentTimestamp(prevDay);
+    fetchDailyStatistics(prevDay);
+  };
+
+  const onPressNextDay = (): void => {
+    const nextDay = getTime(addDays(currentTimestamp, 1));
+    setCurrentTimestamp(nextDay);
+    fetchDailyStatistics(nextDay);
+  };
+
+  useEffect(() => {
+    // When entering the first time
+    if (totalCount === null && !error) {
+      fetchDailyStatistics(Date.now());
+    }
+  });
```

Comment:
Hmm, why is the dependency array excluded here?

Reply:
oops, added this in!